File: hash/완주하지못한선수.py
def solution(participant, completion):
    answer = ''
    num = len(participant)
    participant.sort()
    completion.sort()
    for index in range(num-1):
        if participant[index] != completion[index]:
            answer = participant[index]
            return answer
    answer = participant[num-1]
    return answer

File: hash/완주하지못한선수.js
function solution(participant, completion) {
  var answer = "";
  participant.sort();
  completion.sort();
  for (var i = 0; i < participant.length - 1; i++) {
    if (participant[i] != completion[i]) {
      answer = participant[i];
      return answer;
    }
  }
  answer = participant[participant.length - 1];
  return answer;
}
